docs/reference/builder.md

```diff
@@ -1291,6 +1291,74 @@ The `STOPSIGNAL` instruction sets the system call signal that will be sent to th
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
 
+## HEALTHCHECK
+
+The `HEALTHCHECK` instruction has two forms:
+
+* `HEALTHCHECK [OPTIONS] CMD command` (check container health by running a command inside the container)
+* `HEALTHCHECK NONE` (disable any healthcheck inherited from the base image)
```

Comment:
Indeed, this is one of many reasons that I don't think the health checks should do much more than collect status by default unless the operator who is starting the container explicitly requests that they have some effect on the container's lifecycle -- if that were the case, then inheriting a health check should be mostly harmless (although a way to "reset" and remove the health check would obviously be necessary).